db/migrate/20150629180552_change_password_type_user.rb

```diff
@@ -0,0 +1,6 @@
+class ChangePasswordTypeUser < ActiveRecord::Migration
+  def change
+    remove_column :users, :password
+    add_column :users, :password, :varchar, null: false
```

Comment:
This column name is not very indicative of what you're storing.  Another developer would be highly confused.  Better to name this ```password_hash```.  I'm not really sure why you removed the column and then readded it with basically the exact same definition.  You can use 'changecolumn', but really won't this be problematic for everybody whose password is stored in the plain already in this application?

How would we migrate those users to this new system of password hashing?